GetIntoTeachingApiTests/Controllers/OperationsControllerTests.cs

```diff
@@ -103,5 +105,12 @@ public void LogRequests_IsPresent()
         {
             typeof(OperationsController).Should().BeDecoratedWith<LogRequestsAttribute>();
         }
+
+        [Fact]
+        public void TriggerLocationSync_Authorize_IsPresent()

```

Comment:
Can we add another test that calls the controller action and ensures the job is queued?

Reply:
Had to rework this so that it could be tested properly. Got stuck for a while, turned out I was testing for job type of LocationBatchJob instead of LocationSyncJob.